google/cloud/aiplatform/models.py

```diff
@@ -314,6 +316,266 @@ def deploy(
 
         return endpoint
 
+    def batch_predict(
+        self,
+        job_display_name: str,
+        gcs_source: Optional[Sequence[str]] = None,
+        bigquery_source: Optional[str] = None,
+        instances_format: Optional[str] = "jsonl",
+        gcs_destination_prefix: Optional[str] = None,
+        bigquery_destination_prefix: Optional[str] = None,
+        predictions_format: Optional[str] = "jsonl",
+        model_parameters: Optional[Dict] = None,
+        machine_type: Optional[str] = None,
+        accelerator_type: Optional[str] = None,
+        accelerator_count: Optional[int] = None,
+        starting_replica_count: Optional[int] = None,
+        max_replica_count: Optional[int] = None,
+        labels: Optional[dict] = None,
+        location: Optional[str] = None,
```

Comment:
Should we drop the `location` param since the Model is already associated to a region?

Reply:
Yes, if the service doesn't allow execution of the batch prediction job from a different location than the Model's location.